packages/tuxguitar/PKGBUILD

```diff
@@ -0,0 +1,117 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Stephan Springer <buzo+arch@Lini.de>
+# Contributor: Thanos Apostolou <thanosapostolou@outlook.com>
+# Contributor: Alexandre Moine <alexandre@moine.me>
+# Contributor: Sergej Pupykin <pupykin.s+arch@gmail.com>
+# Contributor: |AhIoRoS| < ahioros@gmail.com >
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+pkgbase=tuxguitar
+pkgname=(tuxguitar tuxguitar-common tuxguitar-gtk2)
+pkgver=1.5.5
+pkgrel=2
+pkgdesc="Multitrack guitar tablature editor and player"
+arch=('x86_64' 'i686' 'armv7h')
+_java_ver=11
+_swt_rel=4.22-202111241800
+_swt_ver=${_swt_rel:0:-13}
+url="https://sourceforge.net/projects/tuxguitar/"
+license=('LGPL')
+depends=('alsa-lib' 'libxtst' 'glibc' "java-runtime-openjdk=${_java_ver}" 'hicolor-icon-theme')
+makedepends=('unzip' 'zip' 'ant' 'jack' 'fluidsynth' "java-environment-openjdk=${_java_ver}" 'maven')
+optdepends=('fluidsynth')
+source=(https://downloads.sourceforge.net/tuxguitar/tuxguitar-$pkgver-src.tar.gz
+        https://archive.eclipse.org/eclipse/downloads/drops4/R-${_swt_rel}/swt-${_swt_ver}-gtk-linux-x86_64.zip
```

Comment:
Why does this package build swt? Can't we just `makedepends` on [swt](https://aur.archlinux.org/packages/swt)? If version 4.21. is too old, `swt` should be updated (I just flagged out-of-date).

Reply:
Well, it's currently an AUR package. Ideally, our binary repo should only have dependencies that are installable from the main Arch repos or itself. What I could of course do is add swt to our repo as well.

Then again, Java / Maven projects often pull in lots and lots of build and runtime dependencies ([check the build log](https://ci.cbix.de/osam-cologne/archlinux-proaudio/49/1/5)) which are distributed with the final binary (in case of runtime deps) so including SWT here is not a too bad practice :)